api/swagger/swagger.json

```diff
@@ -1081,6 +1081,11 @@
                   "type": "string",
                   "format": "date-time",
                   "x-order": 4
+                },
+                "updater_ready": {
+                  "description": "True if pmm-server-upgrade is ready for an update.",
```

Comment:
what does that mean?

Reply:
When this field is `true`, it means the following:
- `pmm-server-upgrade` runs
- It is reachable
- Responds to the `ready` request without an error